gaianet

```diff
@@ -1421,6 +1422,7 @@ show_start_help() {
     printf "Options:\n"
     printf "  --local-only               Start the program in local mode.\n"
     printf "  --base <path>              The base directory of GaiaNet.\n"
+    printf "  --rag                      Force start rag-api-server.\n"
```

Comment:
Please update the description on the `--rag` option:

`Force start rag-api-server even without the "snapshot" field of config.json is empty.\n`